### ISSUE TITLE
TF-TRT example does not convert a graph with INT8 precision

### ISSUE THREAD

**Issue by mumichang** [tensorflow/tensorflow#30619]
System information

- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 18.04
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): 1.14.0
- Python version: 3.6
- Bazel version (if compiling from source): N/A
- GCC/Compiler version (if compiling from source): N/A
- CUDA/cudnn version: 10.0/7.4.1
- GPU model and memory: Titan XP

I simply followed an example in the following directory to generate a resnet (resnet_v1_50) model quantized with INT8

- tensorflow/tensorrt/tftrt/examples/image-classification/image_classification.py

I cannot find any particular error messages in the log. Here is the summary of the graphs.

- num_nodes(native_tf): 741
- num_nodes(tftrt_total): 474
- num_nodes(trt_only): 0
- graph_size(MB)(calib): 97.8
- graph_size(MB)(native_tf): 97.8
- graph_size(MB)(trt): 97.8
- time(s)(saving_frozen_graph): 0.5
- time(s)(trt_calibration): 7.9
- time(s)(trt_conversion): 1.6

The number of nodes in tftrt_total is reduced from native_tf, but cannot see any node of trt_only.
I expected TRTEngineOp in the graph but I cannot see it when I read the cached INT8 graph. Most of data types are DL_FLOAT, not  DL_INT8. 

Based on this observation, I guess the cached graph is not a graph quantized with INT8. 
Any clue why it does not generate a quantized graph? I repeated this for different models such as vgg_16 and inception_v3, but I did not have a luck.

**Comment by mumichang** [comment]
I managed to run it successfully by using nvidia docker container in the following link.
- https://ngc.nvidia.com/catalog/containers/nvidia:tensorflow

Previously I used TensorFlow Docker container. 

I checked the difference between them in terms of their execution behaviors.

- The program, tensorflow/contrib/tensorrt/segment/segment.cc:461] is executed in the nvidia docker container,  but it is not executed in the tensorflow  docker container. 
- The nvidia docker container uses tensorflow 1.13.1, the tensorflow docker containers uses 1.14



Anybody knows why it is so?
